rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -59,6 +60,10 @@ def __init__(
         self.repay_signature = repay_signature
         self.eth_gateways = eth_gateways
         self.label = label
+        self.aave_tokens = GlobalDBHandler().get_evm_tokens(
```

Comment:
@OjusWiZard, I added this part for loading the list of aave v2 or v3 tokens in the constructor, but this is not more needed since you reworked the related _is_aave_token_contract, right?

Reply:
Yeah, it's not needed now. I removed it, but it got back probably when I rebased. I'm removing it now